packages/components/src/Icons/bin/wrapSVGUtils.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-ignore: Redeclared module error
+const svgToComponentTitle = (fileName: string): string => {
+  const split = fileName.split("-")
+  return split
+    .map((word: string) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join("")
+    .replace(".icon", "Icon")
+}
+
+// eslint-disable-next-line no-template-curly-in-string
+const hrefReplacement = "href={`#${uniqueId}`}"
+
+// @ts-ignore: Redeclared module error
+const insertSvgData = (
+  reactTemplate: string,
+  componentName: string,
+  svgContent: string
+): string => {
+  const completedTemplate = reactTemplate
+    .replace("COMPONENT_TITLE", componentName)
+    .replace("SVG_CONTENT", svgContent)
+
+  if (completedTemplate.includes("UNIQUE_ID")) {
+    return completedTemplate
+      .replace('"UNIQUE_ID"', "{uniqueId}")
+      .replace('href="#UNIQUE_ID"', hrefReplacement)
+  } else {
+    // Some raw svgs do not use the use + href pattern.
+    // This removes unused uuids from the template to prevent lint errors.
+    return completedTemplate
+      .replace("const uniqueId = uuidv4()", "")
+      .replace('import { v4 as uuidv4 } from "uuid"', "")
+  }
+}
```

Comment:
Since you already have a return within your `if`, no need for an `else` :)
```suggestion
  }
  
    // Some raw svgs do not use the use + href pattern.
    // This removes unused uuids from the template to prevent lint errors.
    return completedTemplate
      .replace("const uniqueId = uuidv4()", "")
      .replace('import { v4 as uuidv4 } from "uuid"', "")
}
```